### DISCUSSION TITLE
When a new revision should be created

### DISCUSSION THREAD

**Discussion by koksikus** [directus/directus#13321] [Feature Requests]
I still wonder why opening a record without modifying it saves a new revision. 
https://github.com/directus/directus/issues/12838

I understand this is due to the field date update and user update but why is this a new revision. Someone without knowing it will write me a lot of revisions. From what I have noticed, the user often clicks the acceptance button, which makes him save another revision, although only the update date and user field has changed.

If there was a condition that since the change concerns only the date and the user, the revision is not saved until the next field changes, would be great...

**Comment by koksikus** [comment]
Does anyone else also think that a new revision should arise when more fields are changed, and not only the update date ... or I'm just wrong?

**Reply by rijkvanzanten** [reply]
It depends! If you have fields that auto-generate any content, like a timestamp or a UUID, and you submit a change (no matter what the change is), that update should still go through and be tracked as an update, including the revisions. For example `PATCH /items/articles/1` with a body of `{}` should still be an "update", as external influences (like the auto-generated values) can still set field values.

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
